mqtt/mqtt-broker/benches/file_persist_bench.rs

```diff
@@ -0,0 +1,181 @@
+use std::any::type_name;
+use std::collections::{HashMap, VecDeque};
+use std::iter::FromIterator;
+
+use bytes::Bytes;
+use criterion::*;
```

Comment:
Just curious why criterion?

Reply:
We didn't have a benching library and that was the most recommended one when I looked up benchmarking in rust.